### PR TITLE
Fix example query benchmarks

### PR DESCRIPTION
**Pull request by jmarkos**
This works for me, I just did a small change to your work @zhostasa .

### CODE REVIEW

File: plugins/infinispan90/pom.xml [hunk 1]
```diff
@@ -152,7 +152,7 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>9.0.0-SNAPSHOT</version.infinispan>
+            <version.infinispan>9.0.1-SNAPSHOT</version.infinispan>
```
Comment (rvansa) [inline plugins/infinispan90/pom.xml]: 9.0.0.Final is out, you should set it to that one.

File: plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90EmbeddedQueryable.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   public Infinispan90EmbeddedQueryable(Infinispan90EmbeddedService service) {
+      super(service);
+   }
+
+   @Override
+   public Query.Builder getBuilder(String containerName, Class<?> clazz) {
+      return new QueryBuilder90Impl(Search.getQueryFactory(service.getCache(containerName)), clazz);
+   }
+
+   protected static class QueryBuilder90Impl extends QueryBuilder80Impl {
```
Comment (rvansa) [inline plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90EmbeddedQueryable.java]: Could you add a comment saying which signature change this handles?

**Comment by jmarkos** [comment]
Fixed the 2 comments.

**Comment by jmarkos** [comment]
Also rebased, but the continuous benchmark now fails with https://gist.github.com/jmarkos/9f5cb8fba23ee3d5d0e9f4585bd5def2 . I notified Roman, seems to be related to his changes.

**Comment by rvansa** [comment]
This looks good anyway, merging.

**Pull request merged**